CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# Version 0.11.1 (October 2, 2019)
```

Comment:
Definitely should bump version number to `0.12.0` given the large stripe API changes